### ISSUE TITLE
Attach message to ping

### ISSUE THREAD

**Issue by mbwangfpdc** [mterwill/office-hours-help-queue#184]
Currently, the only means of communicating with a student in the queue is to ping them, which is vague, broadcast, which bothers everyone unnecessarily, or to email them, which can be cumbersome. I would like to be able to attach a message when I ping a student.

Right now, I'd primarily use this to remind individual remote students to add online meeting passwords in their help request descriptions. I imagine this feature might also be helpful during in-person office hours, in order to remind students about adding their physical location.

**Comment by mterwill** [comment]
Hey @mbwangfpdc, I like this idea! Would happily accept a PR if you or others are interested.

**Comment by annieli555** [comment]
Hi! My name is Annie Li and I am currently a student in EECS 481 (Software Engineering). For this class, my partner, Phoebe Liang (@pl98 ), and I are working on an assignment that involves contributing to an open-source project. We are very interested in working on some open issues for the EECS OH queue and would love to get started on this and perhaps some others, if they are still needed.

**Comment by mterwill** [comment]
Hi @annieli555 and @pl98, sounds great! Let me know if you have any questions as you're getting started.

**Comment by pl98** [comment]
@mterwill Thanks! We ran into this output from running the setup script: `Not delivering sessions due to an invalid api_key`. How should we go about fixing this?

**Comment by mterwill** [comment]
That should be fine to ignore. The message is from Bugsnag, an error reporting tool we use in production.

**Comment by pl98** [comment]
Hi @mterwill, we're starting to work on this issue this week and ran into some questions. Our first idea was to implement some kind of overlay form using a Modal window from Semantic UI, but we're having some issues with integration. Is there a way to import and use components from libraries like Semantic UI in the current project setup?

**Comment by mterwill** [comment]
👋 we're using an old version of semantic-ui, 2.2.4 (vendored [here](https://github.com/mterwill/office-hours-help-queue/blob/main/vendor/assets/javascripts/semantic.min.js)), and not the React bindings, as those didn't exist when the app was written. You should be able to use any semantic-ui components from that version, though you'd have to manage the modal lifecycle yourself, which could get tricky with React. It's not as pretty, but you could look maybe use [window.prompt](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt) instead? 🤷‍♂️

**Comment by pl98** [comment]
Thanks! I was able to get the modal component working so far, just need to update the ping message correctly before sending the notification.
